src/shared/dom.js

```diff
@@ -22,6 +22,20 @@ export function teardownEach ( iterations, detach, start ) {
 	}
 }
 
+export function spreadAttributes ( node, obj ) {
+	if (obj !== null && typeof obj === 'object') {
+		for ( var property in obj ) {
```

Comment:
What is the policy on browser compatibility?
I personally prefer iterating over Object.keys instead of for-in + hasOwnProperty.

Also, I would personally prefer an early return instead of indenting, but that is a matter of taste :-)